build.gradle

```diff
@@ -146,16 +153,9 @@ def zipSpec = copySpec {
   }
 }
 
-task buildZip(type: Zip) {
+task buildZipSymbols(type: Zip) {
   dependsOn strip
-  archiveBaseName = artifactName
-  with zipSpec
-  destinationDirectory = file("${buildDir}/distributions")
-  archiveVersion = project.version
-  archiveClassifier = artifactClassifier
```

Comment:
There's a complication with changing what the classifier is used for and combining `buildZip` and `buildUberZip` into one task.

Previously a `gradle assemble` in the `ml-cpp` repo (completely independent of Elasticsearch) would result in these files being generated:

```
bash-3.2$ gradle assemble
bash-3.2$ ls -l build/distributions 
total 371512
-rw-rw-r--  1 dave  dev  44297817 Mar  3 10:38 ml-cpp-8.2.0-SNAPSHOT-darwin-x86_64.zip
-rw-rw-r--  1 dave  dev  44297817 Mar  3 10:38 ml-cpp-8.2.0-SNAPSHOT.zip
-rw-rw-r--  1 dave  dev  96359137 Mar  3 10:39 ml-cpp-debug-8.2.0-SNAPSHOT-darwin-x86_64.zip
```

On the PR branch it results in this:

```
bash-3.2$ gradle assemble
bash-3.2$ ls -l build/distributions 
total 284488
-rw-rw-r--  1 dave  dev  96359137 Mar  3 10:33 ml-cpp-8.2.0-SNAPSHOT-debug.zip
-rw-rw-r--  1 dave  dev  44297817 Mar  3 10:32 ml-cpp-8.2.0-SNAPSHOT.zip
```

So we no longer get the platform specific artifacts.  We still need those so that in the full workflow the combine step can download all the platform specific files and combine them.

I don't mind moving where `debug`/`deps`/`nodeps` appears in the file names, so would it be possible to keep the approach of using classifiers for these but still generate files with the platform in their classifiers too, for example, `ml-cpp-8.2.0-SNAPSHOT-darwin-x86_64-debug.zip` where the classifier is `darwin-x86_64-debug`?

Reply:
Ah yeah, I didn't realize the classifier included the current architecture. I've added this back.